plugins/woocommerce/templates/myaccount/form-lost-password.php

```diff
@@ -25,7 +25,7 @@
 	<p><?php echo apply_filters( 'woocommerce_lost_password_message', esc_html__( 'Lost your password? Please enter your username or email address. You will receive a link to create a new password via email.', 'woocommerce' ) ); ?></p><?php // @codingStandardsIgnoreLine ?>
 
 	<p class="woocommerce-form-row woocommerce-form-row--first form-row form-row-first">
-		<label for="user_login"><?php esc_html_e( 'Username or email', 'woocommerce' ); ?></label>
+		<label for="user_login"><?php esc_html_e( 'Username or email', 'woocommerce' ); ?>&nbsp;<span class="required" aria-label="required">*</span></label>
```

Comment:
I guess there are different approaches but IMO instead of using [`aria-label`](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-label) to give the `*` an accessible name, we should instead use `aria-required="true"` on the `<input>` field itself.

The voice reader software should respect that too and indicate the field is required.

What do you think?

Reply:
Thanks for the feedback, yeah there are multiple ways to fix this issue. Though I like the approach your suggested as it better aligns with `a11y` standards. Made change here https://github.com/woocommerce/woocommerce/pull/47229/commits/995928a63327e70230da4d312259ed06ca70a8ca